.pre-commit-config.yaml

```diff
@@ -12,13 +12,13 @@ repos:
         entry: pixi run -e lint end-of-file-fixer
         language: system
         types: [text]
-        stages: [commit, push, manual]
+        stages: [pre-commit, pre-push, manual]
       - id: trailing-whitespace
         name: trailing-whitespace
         entry: pixi run -e lint trailing-whitespace-fixer
         language: system
         types: [text]
-        stages: [commit, push, manual]
+        stages: [pre-commit, pre-push, manual]
```

Comment:
What is the reason for this change?

This way, those two commands will not be installed anymore when running `pre-commit-install-minimal` which is something I'd still like to be included

Reply:
it was complaining to me by my local pre-commit when doing git commit, 
so I think it really doesn't make sense to commit as I don't have this problem with `pixi run lint`.

removing it